set.mm

```diff
@@ -229301,17 +229293,6 @@ U C_ ( T .(+) U ) ) $=
       ( csubg cfv wcel csubmnd co wceq subgsubm smndlsmidm syl ) BCEFGBCHFGBBAI
       BJBCKABCDLM $.
 
-    $( Obsolete proof of ~ lsmidm as of 13-Jan-2023.  Subgroup sum is
```

Comment:
Sorry, my fault: I replaced the proof in one of my latest commits, so the date should have been 13-Jan-2024. Could you add this theorem again (with corrected date), please?